training-maquetado/scss/detail.scss

```diff
@@ -70,19 +82,54 @@
 }
 
 @media only screen and (max-width: $desktop-min-width) {
+  .back-button {
+    margin-bottom: 33px;
+  }
+
   .book {
+    height: 570px;
+    margin-top: 33px;
+    margin: 0;
+    max-width: 854px;
+    padding: 32px;
+  }
+
+  .book-cover {
+    grid-column: 1/1;
+    grid-row: 2/2;
+  }
+
+  .book-details-box {
+    align-self: center;
+  }
+
+  .book-title {
+    align-items: center;
+    display: flex;
     flex-direction: column;
+    grid-column: 2 / 2;
+    justify-self: center;
+    margin: 0;
+
+    &::before {
+      content: none;
```

Comment:
mmm this is kind of weird. you can set display: none if you don't want to show it